we_count/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "we_count",
+  "version": "1.0.0",
+  "description": "We Count Project",
+  "author": "Ted Thompson",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore ."
```

Comment:
This doesn't work. I get the error message:

```
Error: Cannot read .eslintignore file: /Users/ned/Sites/wecount.inclusivedesign.ca/we_count/.gitignore
Error: ENOENT: no such file or directory, open '/Users/ned/Sites/wecount.inclusivedesign.ca/we_count/.gitignore'
```

I think the issue is that you need to have the `.gitignore` at the same level as the Nuxt project. I'd suggest moving everything from `we_count` to the project root to resolve this and make things more straightforward.

Reply:
Having moved `.gitignore` into the `we_count` directory and run this command, I get a number of lint errors. These should be fixed.

Reply:
I structured it so all the front end stuff lives in the `we_count` directory and all the back end stuff lives in the root of the repository. That's why the `.gitignore` is at the root.

I believe it should work leaving the `.gitignore` in the root. It does for me.